forte/processors/third_party/allennlp_processors.py

```diff
@@ -192,6 +195,8 @@ def expected_types_and_attributes(cls) -> Dict[str, Set[str]]:
         :meth:`~forte.pipeline.Pipeline.enforce_consistency` was enabled for
         the pipeline.
         """
-        expectation_dict: Dict[str, Set[str]] = dict()
-        expectation_dict["ft.onto.base_ontology.Sentence"] = set()
+        expectation_dict: Dict[str, Set[str]] = {
+             "ft.onto.base_ontology.Sentence": set()
+        }
+
```

Comment:
I think you can directly return this inline.
```suggestion
return {
             "ft.onto.base_ontology.Sentence": set()
        }
```